File: Foundation/Arrays/max.java
import java.util.Scanner;
public class max
{
    public static Scanner scn = new Scanner(System.in);
    public static void max()
    {
        int n = scn.nextInt();
        int max = (int)-1e8;
        int[] arr = new int[n];
        for(int i = 0; i < arr.length; i++)
        {
            arr[i] = scn.nextInt();
        }

        for(int i = 0; i < arr.length; i++)
        {
            if(arr[i] > max)
            {
                max = arr[i];
            }
        }
        System.out.println(max);
    }

    public static void main(String[] args)
    {
        max();
    }
}

File: Foundation/Arrays/max.cpp
#include<iostream>
#include<vector>
using namespace std;

void max(vector<int>& arr)
{
    int max = -1e8;
    for(int i = 0; i < arr.size(); i++)
    {
        if(arr[i] > max)
        {
            max = arr[i];
        }
    }
    cout<<"Maximun is "<<max;
}

int main()
{
    int n; 
    cin >> n;
    vector<int> arr(n);
    for(int i = 1; i < arr.size(); i++)
    {
        cin>>arr[i];
    }
    max(arr);
    return 0;
}